src/wave.js

```diff
@@ -322,12 +323,13 @@ export class Wave extends mix(WaveBase).with(
         this.drawBoundaries();
         if (this.isDrawBondsEnabled) this.drawBonds();
         this.render();
-        this.createLabelSpritesAsPoints();
+        if (this.fastLabelsRender) this.createLabelsAsPoints();
```

Comment:
Let's have this logic instead a function `createLabels` and abstract it from here